### PR TITLE
Changes for Swift 4.2, Xcode 10 and iOS 12 (beta 1)

### PR DESCRIPTION
**Pull request by markiv**
Please merge into a new branch Swift-4.2.

### CODE REVIEW

**Comment by oks** [comment]
Hi Team! Any updates with this?

File: Example/Example/CustomRows/ImageRow/ImagePickerController.swift [hunk 1]
```diff
@@ -38,10 +38,10 @@ open class ImagePickerController : UIImagePickerController, TypedRowControllerTy
         super.viewDidLoad()
         delegate = self
     }
-    
-    open func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [String : Any]) {
-        (row as? ImageRow)?.imageURL = info[UIImagePickerControllerReferenceURL] as? URL
-        row.value = info[UIImagePickerControllerOriginalImage] as? UIImage
+
+    public func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
```
Comment (mats-claassen) [inline Example/Example/CustomRows/ImageRow/ImagePickerController.swift]: I think this should stay `open`

File: Source/Rows/TextAreaRow.swift [hunk 2]
```diff
@@ -24,6 +24,14 @@
 
 import Foundation
 
+// Temporary workaround for Xcode 10 beta
```
Comment (mats-claassen) [inline Source/Rows/TextAreaRow.swift]: Please add a TODO

**Comment by markiv** [comment]
@mats-claassen Please re-review and merge. Thanks!

File: Source/Rows/TextAreaRow.swift [hunk 3]
```diff
@@ -24,6 +24,14 @@
 
 import Foundation
 
+// TODO: Temporary workaround for Xcode 10 beta
+#if swift(>=4.2)
+import UIKit.UIGeometry
+extension UIEdgeInsets {
+    public static let zero = UIEdgeInsets()
```
[the remark this answers is not in the document]
Comment (mats-claassen) [reply Source/Rows/TextAreaRow.swift]: Fixed

**Pull request merged**